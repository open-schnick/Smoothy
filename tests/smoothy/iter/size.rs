use crate::failing_assertion;
use smoothy::{assert_that, EqualityAssertion, IteratorAssertion};

#[test]
fn size_matches() {
    assert_that([1, 2, 3]).size().is(3);
}

#[test]
fn size_does_not_match() {
    failing_assertion!({
        assert_that([1, 2, 3]).size().is(42);
    });
}
