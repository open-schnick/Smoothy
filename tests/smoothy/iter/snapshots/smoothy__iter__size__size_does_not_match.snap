---
source: tests/smoothy/iter/size.rs
expression: assertion_failed_output
---
assertion failed: `(left == right)`

[1mDiff[0m [31m< left[0m / [32mright >[0m :
[31m<[0m[1;48;5;52;31m3[0m
[32m>[0m[1;48;5;22;32m42[0m
